internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/integration/utils/VeniceServerWrapper.java

```diff
@@ -257,7 +250,7 @@ static StatefulServiceProvider<VeniceServerWrapper> generateService(
             sslFactory,
             regionName);
       } else {
-        VeniceServerWrapper veniceServerWrapper = new VeniceServerWrapper(
+        return new VeniceServerWrapper(
             serviceName,
             dataDirectory,
             null,
```

Comment:
Is there any reason why the `TestVeniceServer` can't be added to the `VeniceServerWrapper` in the else clause? Just wondering what's going on here.

Reply:
I don't remember why we needed `TestVeniceServer` in the first place but I can explain why we can't add it in the `else`.

The `else` block is if we want a "forked" server. By "forked", we mean that it will run in it's own process. So, if we create a `TestVeniceServer`, then that will be running in the same process.